CHANGE_LOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Added Broadcast event `ContactIdNotFound` when using OAuth 2.0 auth code grant
+- Add ability for widget to be draggable
```

Comment:
Is it working for mobile? Let's call it out in the documentation

Reply:
It would work on desktop for now.